install.go

```diff
@@ -732,7 +728,7 @@ func updatePkgbuildSeenRef(bases []Base, cloned stringset.StringSet) error {
 	return errMulti.Return()
 }
 
-func showPkgbuildDiffs(bases []Base, cloned stringset.StringSet) error {
+func showPkgbuildDiffs(bases []Base, cloned stringset.StringSet) {
 	var errMulti multierror.MultiError
```

Comment:
This MultiError is only accumulating errors which are never returned and never treated

Reply:
I kinda forgot those weren't the only two errors that can happen.